### DISCUSSION TITLE
Project Kanban should have sorting functions

### DISCUSSION THREAD

**Discussion by kerthcet** [community/community#28750] [New to GitHub]
Project Kanban should have sorting functions which is helpful.

**Comment by m19c** [comment]
Actually, no... it should not. You should sort your open tasks by importance.
Take JIRA / other boards as an example: you can sort issues and other items in list views but not on the actual board.

**Comment by kerthcet** [comment]
I have no idea whether others meet the same problem, my key pain point is when the column list is quite long, I have to go through the whole items but actually I only care about the most recently updated PRs/Issues.

**Reply by msdev290** [reply]
But I only interested someone who has any development project.
because I'm a Web developer

**Reply by kerthcet** [reply]
I don't know if I understood your answer correctly. I will treat this as another feature named `filtering`.
